app/services/twilio_api_service.rb

```diff
@@ -23,6 +23,14 @@ class TwilioApiService
   delegate :logger, to: Rails
 
   class Error < StandardError
+    STATUS_CODE_REASONS = {400 => :invalid_phone_number}
```

Comment:
Is the sole reason we get 400s? Can we link to the documentation in a comment so that we can revisit this if it changes?

This shouldn't silently hide any client-errors we might hit in the future.